crates/uv/src/main.rs

```diff
@@ -960,10 +960,6 @@ struct PipUninstallArgs {
     #[clap(long, short, group = "sources")]
     requirement: Vec<PathBuf>,
 
-    /// Uninstall the editable package based on the provided local file path.
-    #[clap(long, short, group = "sources")]
-    editable: Vec<String>,
-
```

Comment:
Huh why did we have this in the first place? Is there an actual behavior change here or do we just treat editables like normal packages during uninstall?

We should note this as a breaking change in the changelog.

Reply:
We just treated editables like normal packages (sort of). We removed any packages that matched the URL (regardless of whether they're editable). I guess this probably existed because we didn't support direct URL requirements, so you couldn't do `uv pip uninstall ./path/to/file`. But pip does _not_ allow this. I'll note it as such in the changelog.